heron/scheduler-core/src/java/com/twitter/heron/scheduler/dryrun/JsonFormatterUtils.java

```diff
@@ -0,0 +1,87 @@
+//  Copyright 2018 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.scheduler.dryrun;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import com.twitter.heron.spi.packing.PackingPlan;
+import com.twitter.heron.spi.packing.Resource;
+
+public class JsonFormatterUtils {
+
+  private final ObjectMapper mapper;
+
+  public JsonFormatterUtils() {
+    mapper = new ObjectMapper();
+  }
+
+  public String renderPackingPlan(String topologyName, String packingClass,
+                                      PackingPlan packingPlan) throws JsonProcessingException {
+    ObjectNode topLevel = mapper.createObjectNode();
+    topLevel.put("packingClass", packingClass);
+    topLevel.put("topology", topologyName);
+
+    ArrayNode containers = mapper.createArrayNode();
+
+    for (PackingPlan.ContainerPlan containerPlan : packingPlan.getContainers()) {
+      ObjectNode planNode = mapper.createObjectNode();
+      planNode.put("taskId", containerPlan.getId());
+      containers.add(renderContainerPlan(containerPlan));
+    }
+
+    topLevel.set("containers", containers);
+
+    return mapper.writeValueAsString(topLevel);
+  }
+
+  ObjectNode renderContainerPlan(PackingPlan.ContainerPlan containerPlan) {
+    Resource requiredResources = containerPlan.getRequiredResource();
+    ObjectNode resources = mapper.createObjectNode();
+    resources.put("cpu", requiredResources.getCpu());
+    resources.put("ram", requiredResources.getRam().asBytes());
```

Comment:
I think the output of toString() might be more readable than asBytes() if it works here.

Reply:
This is meant to be machine readable, all of the other formats are human readable. As such I think the raw byte amount would be easier for a machine to parse than having to know the format of `toString()` and adjust accordingly.